xarray/core/datatree_io.py

```diff
@@ -26,18 +27,23 @@ def _datatree_to_netcdf(
     write_inherited_coords: bool = False,
     compute: bool = True,
     **kwargs,
-) -> None:
+) -> None | bytes:
     """This function creates an appropriate datastore for writing a datatree to
     disk as a netCDF file.
```

Comment:
Should this comment be extended to include `bytes` output?

Reply:
I dont' think so -- this function is a private helper, so we don't want to duplicate docs with `DataTree.to_netcdf()`